docs/specs/moniker.yml

```
@@ -1808,3 +1808,184 @@ inputs:
 outputs:
   136a42ac/docs/a.json: |
     {"metadata":{"monikers":["netcore-1.0"]}}
+---
+# Compatible for group in docfx.json of v2
+repos:
+  https://github.com/ops/config-1:
+  - files:
+      .openpublishing.publish.config.json: |
+        {}
+      docfx.json: |
```

consider use YAML to make config easier to read
```suggestion
      docfx.yml: |
```

---

Time-consuming to convert it to `YAML`, will stick to `JSON` here...